<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetUpdate</class>
 <widget class="QWidget" name="WidgetUpdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>381</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Nom à modifier : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="oNom"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Prénom à modifier :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="oPrenom"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>381</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Nouveau nom :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nNom"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Nouveau prénom :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nPrenom"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Nouvelle Entreprise :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nEntreprise"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Nouvelle Photo : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nPhoto"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Nouvel Email : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nEmail"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>379</y>
     <width>761</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="bAnnuler">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bValider">
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
